containerized-data-importer/models/V1beta1CDISpec.ts

```
@@ -0,0 +1,42 @@
+   * @default {}
+   */
+  customizeComponents?: V1Beta1CustomizeComponents;
+  /**
+   * PullPolicy describes a policy for if/when to pull a container image
+   *
+   * Possible enum values:
+   *  - `"Always"` means that kubelet always attempts to pull the latest image. Container will fail If the pull fails.
+   *  - `"IfNotPresent"` means that kubelet pulls if the image isn't present on disk. Container will fail if the image isn't present and the pull fails.
+   *  - `"Never"` means that kubelet never pulls an image, but only uses a local image. Container will fail if the image isn't present
+   */
+  imagePullPolicy?: "Always" | "IfNotPresent" | "Never";
```

ESLint

[error] Replace `"Always"·|·"IfNotPresent"·|·"Never"` with `'Always'·|·'IfNotPresent'·|·'Never'`

(prettier/prettier)